distributed/httpfile.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import requests
```

Comment:
Would you mind making a new PR with just the `s3.read_block` work so that it's easier to merge quickly?